packages/core/theme/src/components/toggle.ts

```diff
@@ -100,7 +100,7 @@ const toggle = tv({
     },
     size: {
       sm: {
-        wrapper: "w-10 h-6 rtl:ml-2 rtl:mr-[unset]",
+        wrapper: "w-10 h-6 me-2",
```

Comment:
The occurance of `me-2` should be removed because they will be adding margin in the end i.e. `mr-2` in case of ltr and `ml-2` in case of rtl.

Reply:
@ShrinidhiUpadhyaya @wingkwong.
Doing this will resolve the issue.